pandas/tests/groupby/test_apply.py

```diff
@@ -710,6 +710,28 @@ def test_apply_with_mixed_types():
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("q", [0.5, [0.25, 0.5, 0.75]])
+def test_groupby_quantile_nullable_integer(q):
```

Comment:
groupby quantile tests appear to be a bit dispersed, however the majority seem to be in tests\groupby\test_function.py . would this be a better location?

Reply:
I think so, went ahead and moved it there